index.html

```diff
@@ -1293,7 +1294,7 @@ <h2>Capability Invocation</h2>
       "capabilityInvocation": [
         <span class="comment">// this method can be used to invoke capabilities as did:...fghi</span>
         "https://controller.example/123456789abcdefghi#keys-1",
```

Comment:
The URI for `capabilityInvocation` has changed from `did:`-scheme to `https:`-scheme, but the comment above says, `invoke capabilities as did:...fghi`. Should that be changed to `invoke capabilities as https:...fghi`?
```suggestion
        <span class="comment">// this method can be used to invoke capabilities as https:...fghi</span>
        "https://controller.example/123456789abcdefghi#keys-1",
```

Reply:
@msporny, @dlongley, @selfissued — Maybe one of you can confirm whether the URI excerpt in the comment should be `did:` scheme or `https:` scheme?